tests/auth.setup.ts

```diff
@@ -0,0 +1,23 @@
+import { test as setup, expect } from '@playwright/test';
```

Comment:
Why the alias here?

Reply:
To make it clear it's not a test, but a setup. Playwright docs suggested same.
https://playwright.dev/docs/auth#basic-shared-account-in-all-tests